docs/AccessControl.md

```diff
@@ -113,6 +113,246 @@ This specifies that the file is expected to be transmitted from the caller to
+handle the others, even if this is properly resolved by DNS. In this
+case it should report this error.
+
+
+The other error codes are HTTP codes adopted for SRT:
+
+### SRT_REJX_BAD_REQUEST
+
+General syntax error. This can be reported in any case when parsing
+the StreamID contents failed, or it cannot be properly interpreted.
+
+
+### SRT_REJX_UNAUTHORIZED
+
+Authentication failed, which makes the client unauthorized to access the
+resource. This error, however, confirms that the syntax is correct and
+the resource has been properly identified. Note that this cannot be
+reported in case when you use a simple user-password authentication
+method because in this case the password is verified only after the
+listener callback handler accepts the connection. This error is rather
+intended to be reported in case of `t=auth` when the authentication
+process has generated some valid session ID, but then the session
+connection has specified a resource that is not within the frames
+of that authentication.
+
+
+### SRT_REJX_OVERLOAD
+
+The server is too heavily loaded to process your request, or you
+have exceeded credits for accessing the service and the resource.
+In HTTP the description mentioned the payment for the service, but
+it was also in use by some service as more general "credit" management
+for a client. In SRT it should be used in case when your service
+is doing any kind of credit management to limit access to the service
+only to some selected clients that "have" enough credit - no matter
+if the credit is something they can recharge themselves or it can
+be also granted them depending on available service resources.
+
+
+### SRT_REJX_FORBIDDEN
+
+Access denied to the resource by any kind of reason. This error is
+independet on an authorization or authentication error (as reported
+by `SRT_REJX_UNAUTHORIZED`), and whichever is more appropriate here is up to
+the decision of the application. This error is usually intended for
+a situation when the resource should be only accessed after a successful
+authorization over a separate auth-only connection, while the query
+in StreamID has correctly specified the resource identity and mode,
+but the session ID (in `s` key) is either not specified or it does
+specify a valid session, however the authorization region for this
+session does not embrace the specified resource.
+
+
+### SRT_REJX_NOTFOUND
+
+The resource specified in the `r` key (with combination of `h` key)
+is not found at this time. This error should be only reported if the
+information about resource accessibility is allowed to be publicly
+visible, otherwise the application might rather report authorization
+errors.
+
+
+### SRT_REJX_BAD_MODE
+
+The mode specified in the `m` key in StreamID is not supported for this request.
+This is in case of having read-only or write-only resources, as well as the
+interactive (bidirectional) access is not predicted as a valid access to this
+resource.
```

Comment:
```suggestion
The mode specified in the `m` key in StreamID is not supported for this request.
This may apply to read-only or write-only resources, as well for when interactive 
(bidirectional) access is not valid for a resource.
```